apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -336,6 +364,9 @@ class OppgaveService(
         }
     }
 
+    // TODO:
+    //  - Kjøre update where sakId  = ?
+    //  - IKKE oppdatere oppgaver som er lukket...?
```

Comment:
Jeg vet ikke hvordan denne blir brukt av forskjellige andre tjenester, men vi bruker vel denne for å bytte til vikafossen blant annet. Så det er kanskje en verdi i å bytte enhet på alle oppgavene, også de ferdigstilte i de tilfellene, og så har vi en historikk i oppgaveendringer

Reply:
Sant nok! Fjerner todoen